books/en_US/src/SUMMARY.md

```diff
@@ -1,10 +1,10 @@
 # Summary
 
-- [Base game](./c01-00-intro.md)
-    - [Project setup](./c01-01-setup.md)
-    - [Entity Component System](./c01-02-ecs.md)
-    - [Components and entities](./c01-03-entities-components.md)
-    - [Rendering system](./c01-04-rendering.md)
+- [游戏基础](./zh_CN/c01-00-intro.md)
```

Comment:
I think we want to revert this file and keep the english version for the english book